dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/EventExporterTest.java

```diff
@@ -590,6 +591,25 @@ void testExportEventsWhenFilteringByNumericDataElements()
     assertContainsOnly(List.of("D9PbzJY8bJM"), events);
   }
 
+  @Test
+  void shouldFilterByEventsWithGivenDataValuesWhenFilterContainsDataElementUIDsOnly()
+      throws ForbiddenException, BadRequestException {
+    EventOperationParams params =
+        EventOperationParams.builder()
+            .eventParams(EventParams.FALSE)
```

Comment:
I think we should make this the default in `EventOperationParams`, do you think so too @muilpp @enricocolasante ? So we fetch as little as possible by default.

Reply:
Agreed.
We have it for enrollments and tracked entities.